docs/index.md

```diff
@@ -1,16 +1,63 @@
 # Charmed OpenSearch Documentation
-The Charmed OpenSearch Operator delivers automated operations management from day 0 to day 2 on the [OpenSearch Community Edition](https://github.com/opensearch-project/OpenSearch/) document database. It is an open source, end-to-end, production-ready data platform product on top of juju.
+[OpenSearch](http://opensearch.org/) is an open-source search and analytics suite. Developers build solutions for search, data observability, 
```

Comment:
Should we be using linebreaks like this, or expecting charmhub/discourse to wrap lines as the screen requires? This might be hard to read on vertical screens such as phones, for instance